man/ctags-lang-verilog.7.rst.in

```diff
@@ -82,7 +82,7 @@ because they cannot be overridden.
 "output.tags"
 with "--options=NONE --sort=no -o - --fields-SystemVerilog=+{parameter} input.sv"
 
-.. code-block:: tags
+.. code-block:: ctags
```

Comment:
This should be "tags" or specifying nothing.

Reply:
fixed.